zircon.core/src/commonTest/kotlin/org/hexworks/zircon/CharacterTilesAsString.kt

```diff
@@ -0,0 +1,34 @@
+package org.hexworks.zircon
+
+import org.hexworks.zircon.api.data.CharacterTile
+import org.hexworks.zircon.api.data.Position
+import org.hexworks.zircon.api.data.Size
+import org.hexworks.zircon.api.data.Tile
+import org.hexworks.zircon.api.graphics.TileComposite
+
+/**
+ * Converts a collection of character tiles (usually found at [TileComposite.tiles]) into a string.
+ *
+ * Note that this is just complicated enough to warrant its own unit tests (in jvmTest).
+ */
+fun TileComposite.characterTilesAsString(padToSize: Size = Size.zero()): String {
```

Comment:
These methods are often named `to*` or `from*` or start with a verb. When you include `As` in the name it usually means a re-wrapping or casting, but this is a conversion. In our case, if you were using the `to` prefix you'd have to name this `toString` which you can't do unfortunately, so maybe a `convertToString` is better? What are your thoughts?

Reply:
Ah I'm usually such a stickler about this particular naming convention...yeah, `convertToString` makes more sense, but also is there room to emphasize that this only works on TileComposites containing character tiles? `convertCharacterTilesToString`?

Reply:
I'm going to go with that then.